Linux Script/configure-linux.sh

```diff
@@ -856,12 +856,13 @@ checkIfTLS()
 						* ) echo "Please answer yes or no.";;
 						esac
 			done			
-	 else
-		    logMsgToConfigSysLog "WARN" "WARN: Your system logs are being send insecurely. We prefer to send system logs securely so switching to secure configuration."
+		else
+       logMsgToConfigSysLog "WARN" "WARN: Your system logs are being send insecurely. We prefer to send system logs securely so switching to secure configuration."
```

Comment:
sent

Reply:
@mchaudhary @mostlyjason, Since this change, was in configure-linux script so I made the above changes in existing PR for configure-linux script here: https://github.com/loggly/install-script/pull/102 so that these changes will not depend on this file-monitoring PR to be merged.